dev/scripts/build.ts

```diff
@@ -1,24 +1,44 @@
+/* eslint-disable max-lines-per-function */
 import { renameSync, rmSync } from "node:fs"
 import { join } from "node:path"
 import { stdout } from "node:process"
+import type { WalkOptions } from "../runtime/exports.js"
 import {
     readFile,
     readJson,
     shell,
+    walkPaths,
     writeFile,
     writeJson
 } from "../runtime/exports.js"
-import { isProd, repoDirs, srcFiles } from "./common.js"
+import { repoDirs, tsFileMatcher } from "./common.js"
+import { denoTransformations } from "./denoBuildTransforms.js"
 import { getProject } from "./docgen/main.js"
 import { mapDir } from "./docgen/mapDir.js"
 import { extractSnippets } from "./docgen/snippets/extractSnippets.js"
 
+const isTestBuild = process.argv.includes("--test")
+
+const isProd = () => process.argv.includes("--prod") || !!process.env.CI
+
+const inFileFilter: WalkOptions = {
+    include: (path) =>
+        tsFileMatcher.test(path) &&
+        /(^src|test|dev\/attest|dev\/runtime)\/?/.test(path),
+    exclude: (path) => /dev\/attest\/test/.test(path),
```

Comment:
We could just add the negated version of this to the include option and get rid of `exclude`.